force-app/tdtm/classes/TDTM_TriggerHandler.cls

```diff
@@ -279,6 +281,28 @@ public class TDTM_TriggerHandler {
         }
     }
     
+    /*******************************************************************************************************
+    * @description Performs pending Errors and DML operations, on the records stored in the DMLWrapper that it gets as parameter.
+    * We want this method to be static so that it can be called from the future method in TDTM_Runnable.
+    * @param dmlWrapper The class that contains the records that need to have DML performed on.
+    * @return void
+    */
+    public static void processDMLWithErrors(TDTM_Runnable.DmlWrapper dmlWrapper) {
+        processDMLAllOrNothing(dmlWrapper, true);
+        processDMLError(dmlWrapper);
+    }
+
+    public static void processDMLError(TDTM_Runnable.DmlWrapper dmlWrapper) {
+        if (dmlWrapper.objectsWithError?.isEmpty()) {
+            return;
+        }
+
+        preventCachingFalseTransaction = true;
```

Comment:
The prevent caching flag will only be set in the `processDMLWithErrors` and will not interfered with original `processDML` method

Reply:
I like that you left this open for use by other objects (if we were to change their `processDML` calls) 👍